Sources/PackageType/Module.swift

```diff
@@ -99,6 +99,28 @@ public class TestModule: SwiftModule {
     }
 }
 
+public class XcodeModule: Module {
```

Comment:
I tried to make a protocol. But -G use all Module fields. I made a protocol which copies all Module properties and stopped in 
```swift
public var dependencies: [Module]  /// in build order
```
Don't remember an error. Also I'm not very good in swift, maybe it possible.